numpy/linalg/lapack_lite/blas_lite.c

```diff
@@ -16,6 +16,15 @@ extern doublereal dlamch_(char *);
 
 extern doublereal dlapy2_(doublereal *x, doublereal *y);
 
+/*
```

Comment:
Note that right now all the generated code goes through a comment transformation script that makes them look like the one I wrote